packages/shared/src/components/search/SearchContainer.tsx

```diff
@@ -16,27 +20,40 @@ export function SearchContainer({
   isLoading,
   chunk,
 }: SearchContainerProps): React.ReactElement {
+  const content = chunk?.response || '';
+
   return (
-    <main className="m-auto w-full">
-      <div className="grid grid-cols-1 laptop:grid-cols-3 gap-y-6 py-8 m-auto w-full max-w-screen-laptopL">
-        <main className="flex flex-col flex-1 col-span-2 px-4 laptop:px-8">
-          {isLoading ? (
-            <>
-              <Pill className="!h-16" />
-              <Pill className="mt-3 !h-2" />
-            </>
-          ) : (
-            <SearchBar
-              onSubmit={onSubmit}
-              chunk={chunk}
-              showProgress
-              isLoading={isLoading}
-            />
-          )}
-        </main>
-        <SearchFeedback />
+    <div className="grid grid-cols-1 laptop:grid-cols-3 gap-y-6 py-8 m-auto w-full max-w-screen-laptopL">
+      <main className="flex flex-col col-span-2 px-4 laptop:px-8">
+        {isLoading ? (
+          <>
+            <Pill className="!h-16" />
+            <Pill className="mt-3 !h-2" />
+          </>
+        ) : (
+          <SearchBar
+            onSubmit={onSubmit}
+            chunk={chunk}
+            showProgress
+            isLoading={isLoading}
+          />
+        )}
         {children}
-      </div>
-    </main>
+      </main>
+      <PageWidgets tablet={false} className="gap-6 items-center !px-0">
+        <SearchFeedback className="order-2 laptop:order-1 max-w-widget" />
+        {!!content && (
+          <SearchSourceList
+            className={classNames(
+              'order-1 laptop:order-1',
+              chunk?.status && 'laptop:mt-6 laptop:pt-0.5',
```

Comment:
The only place where I feel it is a bit hacky is the spacing we apply here on laptop. Though imo, it is better than rendering three same elements and placing conditions on each instance.